Portal/vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/ConfigurationBusiness.java

```diff
@@ -106,8 +106,6 @@ public boolean validateSession(String email, String session) throws BusinessExce
             if (email != null && session != null) {
                 UserDAO userDAO = CoreDAOFactory.getDAOFactory().getUserDAO();
                 if (userDAO.verifySession(email, session) && !userDAO.isLocked(email)) {
-                    String newSession = UUID.randomUUID().toString();
-                    userDAO.updateSession(email, newSession);
```

Comment:
The reason for this is that session ids can only be used once, therefore if someone intercepts the session id he/she couldn't use it to log in again. What is the motivation for removing it?